CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.4.9 (2023-07-12)
+### Added
+- SDK: Support swift 5.8.1
```

Comment:
@spvagelis 
Just make this ` - Support for Swift 5.8.1`. Without the `SDK: `.

Reply:
Done